catalog/iam/iam-user-unused-credentials-check.yaml

```diff
@@ -0,0 +1,13 @@
+iam-user-unused-credentials-check:
+  identifier: IAM_USER_UNUSED_CREDENTIALS_CHECK
+  description:
+    Checks whether your AWS Identity and Access Management (IAM) users have passwords or active access keys that have
+    not been used within the specified number of days you provided. Re-evaluating this rule within 4 hours of the first
+    evaluation will have no effect on the results.
+  inputParameters:
+    maxCredentialUsageAge: 90
```

Comment:
I'm not sure how to handle these type of params to make them user-configurable.

Is there some way to look this up in the stack and if not specified there, default to this hardcoded value?

Reply:
all those YAML files are templates.
When we use our `terraform-yaml-config` module, the values in the templates are replaced automatically.

https://github.com/cloudposse/terraform-aws-service-control-policies/blob/master/examples/complete/main.tf

https://github.com/cloudposse/terraform-yaml-config/blob/main/main.tf#L31

Reply:
use https://www.terraform.io/docs/configuration/functions/templatefile.html
add `var.parameters` with some default values (which user could override)

Reply:
I'm not sure if that would work with a large number of parameters and they are completely arbitrary per config rule. 

For example, the one above has a single param `maxCredentialUsageAge" and others have lots of different params:

https://github.com/cloudposse/terraform-aws-config/pull/4/files#diff-ba592a9fad5cfa7aa159dca2cd560bb580226988c89f9011d7e2bbb8a428bb2e

Reply:
The use can/should maybe fork the rules that need to be customized and have local ones?